pom.xml

```diff
@@ -77,8 +77,8 @@ flexible messaging model and an intuitive client API.</description>
   </issueManagement>
 
   <properties>
-    <maven.compiler.source>17</maven.compiler.source>
-    <maven.compiler.target>17</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
This would be bringing everything back to Java 11. The goal of PIP-156 was to get brokers on Java 17.

Reply:
I sent a message on the ML.
I believe it is too strong to require people moving to JDK17 without having forced them switch to JDK11.

I know few companies (especially Banks and Insurance Companies for instance) that are not able to switch so easily the JDK, especially if you were running on JDK8.
I am not talking about Docker, but clusters on traditional/legacy VMs